Owncloud iOs Client/Network/Downloads/Download.m

```diff
@@ -669,6 +672,9 @@ - (void) updateThisEtagWithTheLast {
                             case kOCErrorServerPathNotFound:
                                 [weakSelf.delegate downloadFailed:NSLocalizedString(@"download_file_exist", nil) andFile:weakSelf.fileDto];
                                 break;
+                            case kOCErrorServerMaintenanceError:
+                                [weakSelf.delegate downloadFailed:NSLocalizedString(@"maintenance_mode_on_server_message", nil) andFile:weakSelf.fileDto];
```

Comment:
Buff, it's different I have to keep this kind of implementation because the download case is a delegate class that it's used in some others classes that have their own methods to manage this errors.